src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -371,6 +410,147 @@ private static Properties readConfigFileProperties(final Path configFilePath) {
         return configProperties;
     }
 
+    /**
+     * Logs the meta data about the data sources from the given {@code dataSourcesPath}.
+     * We assume the data sources path is OK in the case that the version information cannot be read because the
+     * user can create their own data sources directory, which may not contain the metadata we seek.
+     * @param dataSourcesPath {@link Path} to a Data Sources directory to check.
+     * @return {@code True} if the given {@code dataSourcesPath} is equal to or newer than the minumum version, or if the version is unreadable.  {@code False} otherwise.
+     */
+    private static boolean logDataSourcesInfo(final Path dataSourcesPath) {
+
+        boolean dataSourcesPathIsAcceptable = true;
+
+        final Path readmePath = dataSourcesPath.resolve(IOUtils.getPath(README_FILE_NAME));
+
+        String version = null;
+
+        if ( Files.exists(readmePath) && Files.isRegularFile(readmePath) && Files.isReadable(readmePath) ) {
+
+            try ( final BufferedReader reader = Files.newBufferedReader(readmePath) ) {
+
+                Integer versionMajor   = null;
+                Integer versionMinor   = null;
+                Integer versionYear    = null;
+                Integer versionMonth   = null;
+                Integer versionDay     = null;
+                String source          = null;
+                String alternateSource = null;
+
+                // Get the info from our README file:
+                String line = reader.readLine();
+                while ((line != null) && ((version == null) || (source == null) || (alternateSource == null))) {
+
+                    if (version == null && line.startsWith(README_VERSION_LINE_START)) {
```

Comment:
Why have a README and a config file?  Can't we merge those?

Reply:
I don't understand... there is no config file for the data sources directory - only a README file.

Reply:
Ok... I thought this code was being run for each datasource.  Not the datasources as a whole...